src/simd_256_u16.h

```diff
@@ -138,7 +137,7 @@ inline m256i mul(m256i a, m256i b, aint16 card)
     return mul(a, b);
 }
 
-/** Negatize elements of buffers
+/** Apply an element-wise negation to a buffer
  */
 inline void neg(size_t len, aint16* buf, aint16 card = F3)
 {
```

Comment:
maybe it's just me, but I think the variable could have clearer names:
- `_buf` => `chunks_buf` (cast the buffer of item into a buffer of chunks (of N items))
- `ratio` => `chunks_sz` (size of a chunk, in number of items)
- `_len` => `nb_chunks` (number of chunk)
- `_last_len` => `nb_trailing` (number of trailing items)

Reply:
These names are used similarly in other places so I keep them